pac4j-core/src/test/java/org/pac4j/core/authorization/authorizer/IsAuthenticatedAuthorizerTests.java

```diff
@@ -74,6 +74,6 @@ public void testAnonymousProfileRedirectionUrl() {
         profiles.add(new AnonymousProfile());
         ((IsAuthenticatedAuthorizer) authorizer).setRedirectionUrl(PAC4J_URL);
         TestsHelper.expectException(() -> authorizer.isAuthorized(MockWebContext.create(), profiles), HttpAction.class,
```

Comment:
by the way, you could use https://github.com/junit-team/junit4/wiki/Exception-testing for expecting exceptions in tests!

Reply:
Interesting, even I found my `TestsHelper` more convenient...